configuration.go

```diff
@@ -56,4 +56,10 @@ type ClientConfiguration struct {
 	// TokenCache is used to retrieve previously acquired tokens and saving new ones
 	// If not set we do not use a cache to store the new acquired tokens
 	TokenCache authenticator.TokenCache
+	// TokenExpiration defines lifetime of generated tokens
+	// Has no effect for tokens provided via the Token field
```

Comment:
I would change the `Has no effect...` comment to:
```
// This is used by the authenticator upon requesting new tokens, this has no effect for statically provided tokens
```

Reply:
I disagree with the 2dn part of tour suggested phrase, but will change if needed.

"statically provided tokens" is a term that is references nowhere, so a user would have to guess what it means, whereas "tokens provided via the Token field" has a direct reference to something in the documentation, directly above it for that matter.

Reply:
Agreed :+1:

Reply:
I rephrased the comments